lib/davinci_pas_test_kit/generated/v2.0.1/pas_server_pended_use_case_group.rb

```diff
@@ -60,9 +60,14 @@ def use_case
       end
       
       group do
-        title 'Server can respond to claims submitted for inquiry'
+        title 'Server can notify client of updates and respond to claims submitted for inquiry'
         
         test from: :prior_auth_claim_response_update_notification_validation
+        test from: :subscriptions_r4_server_notification_conformance do
+          config options: { subscription_type: 'id-only' }
+          verifies_requirements 'hl7.fhir.us.davinci-pas_2.0.1@145'
```

Comment:
run `bundle exec rake requirements:generate_coverage` to add these to the requirements coverage document - I don't see that updated yet.

Reply:
https://github.com/inferno-framework/davinci-pas-test-kit/pull/31/commits/32c7b075e0393b69a852b170c852ef66d6c7f541